dts/arm/nxp/nxp_imx95_m7.dtsi

```diff
@@ -6,6 +6,7 @@
 
 #include <arm/armv7-m.dtsi>
 #include <zephyr/dt-bindings/clock/imx95_clock.h>
+#include <zephyr/dt-bindings/power/imx95_power.h>
```

Comment:
I don't see any `macro`  from this header used. Is this needed?

Reply:
Let me remove it. It's true IP node hadn't used it now.
It's used in soc.c for power init now.